grouper-parent/pom.xml

```diff
@@ -37,7 +37,7 @@
         <module>../grouper-ui</module>
         <module>../grouper-ws</module>
         <module>../grouper-misc/grouper-installer</module>
-        <module>../grouper-misc/webapp/grouper-ui-webapp</module>
+        <module>../grouper-misc/webapp</module>
```

Comment:
Recommend removing this. This is the list of modules that are built and published at every release, and for which documentation is built. You can always build this manually, even if it's not in this list.